idd/Energy+.idd.in

```diff
@@ -8947,7 +8947,6 @@ ConstructionProperty:InternalHeatSource,
        \note uniform spacing between tubes or resistance wires in direction
        \note perpendicular to main intended direction of heat transfer
   N5;  \field Two-Dimensional Temperature Calculation Position
-       \required-field
```

Comment:
I think that makes sense, @mjwitte do you agree?

Reply:
> I think that makes sense, @mjwitte do you agree?

Yes.